umbraco/Umbraco-CMS
# V8: AB 2438 - RTE Pasted/Dragged Images Media Folder Config

## Note
This branch has started work from the branch/PR https://github.com/umbraco/Umbraco-CMS/pull/6246
~So if there any changes or re-review done it will need to be synced up in here too~
**Already done & merged up into this branch a couple of times**

## Feature
This adds a datatype configuration to the RTE datatype and the Grid to allow you to configure where the pasted/dragged images into the RTE will get saved in the media library.

Fixes [AB#2438](https://umbraco.visualstudio.com/243e7927-03b2-44e2-908f-d4ac7ea5daaa/_workitems/edit/2438)

## Test Notes
* Upload an image as before in the RTE from dragging or pasting an image in
* With no Folder picked these will go into the root of the media library
* Go back & configure the property editor to choose a folder you wish to save them in
* Repeat process & verify media saved/created in chosen folder

* Rinse & repeat for Nested Content with RTE in it
* Rinse & repeat for the Grid with one or more cells that are RTE's
